covert/__main__.py

```diff
@@ -109,11 +110,20 @@ def argparse():
   elif next((k for k, v in encargs.items() if av[0] in v), None) is not None:
     args.mode, ad, modehelp = 'enc', encargs, enchelp
     av.insert(0, None)
```

Comment:
This case no longer works. It was intended to choose enc mode if the first argument is was in encargs, even if no mode was specified. You can remove these lines, we don't need that feature.